release.hcl

```diff
@@ -0,0 +1,15 @@
+group "release" {
```

Comment:
do you mind keeping this file under .github/workflow for now since it's only been used in that context? Thanks!